main.go

```diff
@@ -60,6 +63,8 @@ const (
 	MYSQLDUMP string = "mysqldump"
 	// MYSQL is the driver name for MySQL.
 	MYSQL string = "mysql"
+	// DYNAMODB is the driver name for AWS DynamoDB.
```

Comment:
Maybe update comment:

// DYNAMODB is the driver name for AWS DynamoDB.
// This is an experimental driver; implementation in progress.